create-release.py

```diff
@@ -610,6 +610,13 @@ def prepare_extensions_release() -> None:
             f"and execute them against local or cloud-based resources."
             f"See https://elyra.readthedocs.io/en/{config.new_version}/user_guide/enhanced-script-support.html",
         ),
+        "elyra-scala-editor-extension": SimpleNamespace(
+            packages=["scala-editor-extension", "metadata-extension", "theme-extension"],
+            description=f"The Scala Language editor extension contains support for Scala files, which can take "
+            f"advantage of the Hybrid Runtime Support enabling users to locally edit .scala files "
+            f"and execute them against local or cloud-based resources."
+            f"See https://elyra.readthedocs.io/en/{config.new_version}/user_guide/enhanced-script-support.html",
```

Comment:
See https://github.com/elyra-ai/elyra/pull/2860; the URL is invalid

```suggestion
            f"See https://elyra.readthedocs.io/en/v{config.new_version}/user_guide/enhanced-script-support.html",
```